source/meshconv/meshconv.cpp

```diff
@@ -14,28 +65,75 @@ static int mainInternal( int argc, char **argv )
     float targetEdgeLen = 0;
 
     namespace po = boost::program_options;
-    po::options_description desc( "Available options" );
-    desc.add_options()
+    po::options_description generalOptions( "General options" );
+    generalOptions.add_options()
         ("help", "produce help message")
         ("input-file", po::value<std::filesystem::path>( &inFilePath ), "filename of input mesh")
-        ("output-file", po::value<std::filesystem::path>( &outFilePath ), "filename of output mesh")
-        ("remesh", po::value<float>( &targetEdgeLen )->implicit_value( targetEdgeLen ), "optional argument if positive is target edge length after remeshing")
+        ("output-file,o", po::value<std::filesystem::path>( &outFilePath ), "filename of output mesh")
+        ;
+
+    po::options_description commands( "Commands" );
+    commands.add_options()
+        ( "remesh", po::value<float>( &targetEdgeLen )->implicit_value( targetEdgeLen ), "optional argument if positive is target edge length after remeshing" )
+        ( "unite", po::value<std::filesystem::path>(), "unite mesh from input file and given mesh" )
+        ( "subtract", po::value<std::filesystem::path>(), "subtract given mesh from input file mesh given mesh" )
+        ( "intersect", po::value<std::filesystem::path>(), "intersect mesh from input file and given mesh" )
```

Comment:
Do we need it if commands are parsed manually now?

Reply:
yes. it give information that key should have value